raddb/all.mk

```diff
@@ -167,46 +149,20 @@ ${top_srcdir}/raddb/certs/passwords.mk: $(wildcard ${top_srcdir}/raddb/certs/*cn
 #  done with the CI environment's caching features and not committed to the
 #  git repository.
 #
-define BUILD_CERT
-${1}/${2}/${3}.key: ${1}/${3}.cnf ${1}/passwords.mk | ${1}/${2}
-	$${Q}echo CERT-KEY ${2}/${3}
-	$${Q}$$(MAKE) -C ${1} ${2}/${3}.key
-	@touch $$@
-
-${1}/${2}/${3}.csr: ${1}/${2}/${3}.key
-	$${Q}echo CERT-CSR ${2}/${3}
-	$${Q}$$(MAKE) -C ${1} ${2}/${3}.csr
-	@touch $$@
-
-${1}/${2}/${3}.pem: ${1}/${2}/${3}.key
-	$${Q}echo CERT-PEM ${2}/${3}
-	$${Q}$$(MAKE) -C ${1} ${2}/${3}.pem
-	@touch $$@
-
-${1}/${2}/${3}.crt: ${1}/${2}/${3}.pem
-	$${Q}echo CERT-CRT ${2}/${3}
-	$${Q}$$(MAKE) -C ${1} ${2}/${3}.crt
-	@touch $$@
-
-ifneq "${3}" "ca"
-#  client, server, and OCSP certs need the CA cert.
-${1}/${2}/${3}.crt: ${1}/${2}/ca.crt
-
-${1}/${2}/${3}.crt: ${1}/${2}/${3}.csr
-endif
-
-endef
-
-#
-#  Generate local certificate products when doing a non-package
-#  (i.e. developer) build.
+#  To avoid race conditions when calling `openssl ca` certificates are built
+#  in a specific sequence.
 #
-$(foreach dir,rsa ecc,$(foreach file,ca server client ocsp,$(eval $(call BUILD_CERT,${top_srcdir}/raddb/certs,${dir},${file}))))
+build.raddb: ${top_srcdir}/raddb/certs/ecc/ocsp.pem
 
-${top_srcdir}/raddb/certs/dh: ${top_srcdir}/raddb/certs/passwords.mk
-	${Q}echo CERT-DH $@
-	${Q}$(MAKE) -C ${top_srcdir}/raddb/certs/ $(notdir $@)
-	${Q}touch $@
+${top_srcdir}/raddb/certs/ecc/ocsp.pem:
+	${Q}echo Making certs for $@
+	${Q}$(MAKE) -C ${top_srcdir}/raddb/certs/ dh
```

Comment:
I think you can just make all?

Reply:
That does a few other bits, which aren't required for CI, and I think also had other parallel run issues

Reply:
It won't have parallel run issues because by default Make only builds one target at a time.

Reply:
The issues were with the .vrfy targets in raddb/certs/Makefile - they did not have all their dependencies listed